paper-radio-button.html

```diff
@@ -0,0 +1,162 @@
+<!--
+Copyright (c) 2015 The Polymer Project Authors. All rights reserved.
+This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
+The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
+The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
+Code distributed by Google as part of the polymer project is also
+subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
+-->
+
+<!--
+`paper-radio-button` is a button that can be either checked or unchecked.
+User can tap the radio button to check it.  But it cannot be unchecked by
+tapping once checked.
+
+Use `paper-radio-group` to group a set of radio buttons.  When radio buttons
+are inside a radio group, only one radio button in the group can be checked.
+
+Example:
+
+    <paper-radio-button></paper-radio-button>
+
+Styling a radio button:
+
+<style is="x-style">
+  * {
+    /* Unhecked state colors. */
+    --paper-radio-button-unchecked-color: #5a5a5a;
+    --paper-radio-button-unchecked-ink-color: #5a5a5a;
+
+    /* Checked state colors. */
+    --paper-radio-button-checked-color: #009688;
+    --paper-radio-button-checked-ink-color: #0f9d58;
+  }
+</style>
+
+@group Paper Elements
+@element paper-radio-button
+-->
+
+<link href="../polymer/polymer.html" rel="import">
+<link rel="import" href="../paper-ripple/paper-ripple.html">
+
+<dom-module id="paper-radio-button">
+  <style is="x-style">
+    * {
+      --paper-radio-button-unchecked-color: #5a5a5a;
```

Comment:
Use the colors defined in `x-theme.html` from PolymerLabs/paper-styles (maybe the `--default-primary-color`?) so all the elements have a unified theme?